internal/interfaces/grpc/handler/transport.go

```diff
@@ -0,0 +1,33 @@
+package grpchandler
+
+import (
+	"context"
+
+	"github.com/tdex-network/tdex-protobuf/generated/go/transport"
+)
+
+type transportHandler struct {
+	transport.UnimplementedTransportServer
+}
+
+func NewTransportHandler() transport.TransportServer {
+	return newTransportHandler()
+}
+
+func newTransportHandler() *transportHandler {
+	return &transportHandler{}
+}
+
+func (t transportHandler) SupportedContentTypes(
+	context.Context,
+	*transport.SupportedContentTypesRequest,
+) (*transport.SupportedContentTypesReply, error) {
+	return &transport.SupportedContentTypesReply{
+		AcceptedTypes: []transport.ContentType{
+			transport.ContentType_JSON,
```

Comment:
Remove. At the moment the daemon does not support HTTP1 requests

Reply:
Done